.github/workflows/release-discover-branches.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         with:
           repository: 'konflux-ci/rpm-lockfile-prototype'
           path: ./src/github.com/openshift-knative/rpm-lockfile-prototype
-          ref: '283ee2cd7938a2142d8ac98de33ba0d0e3ac146f'
+          ref: '6d222512f5e7b89c59fdcf9bddc9f4d535f3f255'
```

Comment:
Maybe we can change it to ref: `main`

Reply:
Updated to tag v0.15.0 which has the fix. This should also help us to keep the build reproduceable until it is more stable